WalletWasabi.Gui/Controls/WalletExplorer/TransactionInfoTabView.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity"
+             xmlns:behaviors="clr-namespace:WalletWasabi.Gui.Behaviors;assembly=WalletWasabi.Gui"
+             xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui"
+             xmlns:converters="clr-namespace:WalletWasabi.Gui.Converters;assembly=WalletWasabi.Gui"
+             x:Class="WalletWasabi.Gui.Controls.WalletExplorer.TransactionInfoTabView">
+  <UserControl.Resources>
+    <converters:BooleanStringConverter x:Key="BooleanStringConverter" />
+    <converters:LurkingWifeModeStringConverter x:Key="LurkingWifeModeStringConverter" />
+  </UserControl.Resources>
+  <controls:GroupBox BorderThickness="0" Classes="docTabContainer">
+    <controls:GroupBox Title="Transaction Details" TextBlock.FontSize="16" Padding="30" Margin="0 10 0 0">
+      <StackPanel DataContext="{Binding Transaction}">
+        <Grid ColumnDefinitions="170, *">
+          <TextBlock Classes="monospaceFont" Text="Amount Transacted:" />
+          <controls:ExtendedTextBox Grid.Column="1" Classes="selectableTextBlock monospaceFont Transparent" Text="{Binding AmountBtc}" />
+        </Grid>
+        <Grid ColumnDefinitions="170, *">
+          <TextBlock Classes="monospaceFont" Text="Transaction Id:" />
+          <controls:ExtendedTextBox Grid.Column="1" Classes="selectableTextBlock monospaceFont Transparent" Text="{Binding TransactionId, ConverterParameter=50, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" />
+        </Grid>
+        <Grid ColumnDefinitions="170, *">
+          <TextBlock Classes="monospaceFont" Text="Transaction Date:" />
+          <controls:ExtendedTextBox Grid.Column="1" Classes="selectableTextBlock monospaceFont Transparent" Text="{Binding DateTime}" />
+        </Grid>
+        <Grid ColumnDefinitions="170, *">
+          <TextBlock Classes="monospaceFont" Text="Block Height:" />
+          <controls:ExtendedTextBox Grid.Column="1" Classes="selectableTextBlock monospaceFont Transparent" Text="{Binding BlockHeight}" />
+        </Grid>
+        <Grid ColumnDefinitions="170, *">
+          <TextBlock Classes="monospaceFont" Text="Confirmed:" />
+          <controls:ExtendedTextBox Grid.Column="1" Classes="selectableTextBlock monospaceFont Transparent" Text="{Binding Confirmed, Converter={StaticResource BooleanStringConverter}, ConverterParameter=Yes:No, Mode=OneWay}" />
+        </Grid>
```

Comment:
IMO we should remove this, because we have  `Confirmations` and when it is > 0 then it is confirmed.

Reply:
There is space and for some users, it can be helpful. I would not touch it.